src/api.c

```diff
@@ -210,7 +255,7 @@ API int seccomp_reset(scmp_filter_ctx ctx, uint32_t def_action)
 	if (ctx == NULL || db_col_action_valid(NULL, def_action) < 0)
 		return -EINVAL;
```

Comment:
Do you want to call `_rc_filter()` here as well?

Reply:
Good catch, thanks.  I did a second check and caught a few others in src/api.c.  This should be fixed in the next force push.